src/NativeScript/GlobalObject.moduleLoader.mm

```diff
@@ -154,7 +159,7 @@
             absolutePath = [static_cast<NSString*>(self->applicationPath()) stringByAppendingPathComponent:@"app/tns_modules/tns-core-modules"];
             isModuleRequire = true;
         }
-        absolutePath = [normalizePath([absolutePath stringByAppendingPathComponent:path]) stringByStandardizingPath];
+        absolutePath = normalizePath([absolutePath stringByAppendingPathComponent:path]);
```

Comment:
The call to `normalizePath` seems redundant here. In fact I think that the `normalizePath` function is redundant and I don't see it doing anything more than `stringByStandardizingPath` which we always call.